src/index.js.mustache

```diff
@@ -51,7 +51,7 @@ export const ReactSelector = Selector(selector => {
     if(foundDOMNodes)
         return foundDOMNodes;
 
-    throw new Error('testcafe-react-selectors supports React version 15.x and newer');
+    throw new Error("testcafe-react-selectors supports React version 15.x and newer. To wait until the React's component tree is loaded, add the `waitForReact` method to fixture's `beforeEach` hook.");
```

Comment:
hm... Now this error is raised in two different occurrences.

It's better if we could have two different errors, but if it's not possible - let's make the message more clear.

```
    throw new Error("React component tree is not loaded yet or the current React version is not supported. This module supports React version 15.x and newer. To wait until the React's component tree is loaded, add the `waitForReact` method to fixture's `beforeEach` hook.");
```